wolox-books/src/styles.scss

```diff
@@ -1 +1,2 @@
-/* You can add global styles to this file, and also import other style files */
+@import './scss/equalizer';
+@import './app/register/register.component';
```

Comment:
If you add this to the styles of the component, there's no need to add it in here